sdk/eventhub/event-hubs/samples/browserSample/README.md

```diff
@@ -0,0 +1,81 @@
+---
+
+# Azure Event Hubs client library samples for JavaScript
+
+This sample programs show how to use the JavaScript client libraries for Azure Event Hubs to send and receive events in the browser while authenticating with Azure Active Directory.
+
+| **File Name**                     | **Description**                                                                                                                         |
+| --------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| [sendEvents.js][sendevents]       | Demonstrates how the send() function can be used to send events to an Event Hub instance.                                               |
+| [receiveEvents.js][receiveevents] | Demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance. |
+| [configuration.js][configuration] | Contains the configuration needed to authenticate and connect to an Event Hub instance.                                                 |
+| [index.js][app]                   | Hooks up the send and receive samples to their respective buttons in the web application.                                               |
+| [index.html][htmlpage]            | The web page that loads and runs the samples. Use a local web service to test by running `npm start` after building the app.            |
+
+## Prerequisites
+
+The samples are compatible with Node.js >= 8.0.0 and run in browsers that support async/await (e.g. Edge, Firefox, Chrome.)
+
+You need [an Azure subscription][freesub] and [an Azure Event Hub resource][azhubacct] to run these sample programs.
+Samples retrieve credentials using the [InteractiveBrowserCredential][browsercred] from `@azure/identity`.
+You can set the Event Hubs and authentication properties you'll need to run the sample in the [configuration.js][configuration] file.
+
+Register a new application in AAD and assign the "Azure Event Hubs Data Owner" role to it.
+
+- See https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app
+  to register a new application in the Azure Active Directory.
+- Note down the client id and tenant id from the above step.
+  You will need to set these in the [configuration.js][configuration] file.
+
+Ensure your app registration has been configured properly to allow enable the [implicit grant flow][implicitgrantflow]
+and allow `Access tokens` and `ID tokens` to be issued by the authorization endpoint.
+In your app registration, you will also need to add a permission for the `Microsoft.EventHubs` app.
+When adding permission for `Microsoft.EventHubs`, the type should be `delegated permissions` and the permission should be `user_impersonation`.
+
+## Setup
+
+To run the samples using the published version of the package:
+
+1. Install the dependencies using `npm`:
+
+```bash
```

Comment:
Your discrimination against powershell users continues unabated.

Reply:
The funny thing is I ran these commands in powershell :smile: